### PR TITLE
add tests, tox config, and better item template

### PR DESCRIPTION
**Pull request by DylannCordel**

### CODE REVIEW

**Comment by weber-s** [comment]
LGTM.

TODO after #3 : reformat black with line length = 99

File: dalec_caldav/proxy.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
+from datetime import date, datetime, timedelta
```
Comment (weber-s) [inline dalec_caldav/proxy.py]: isort / black to redo after #3

File: setup.cfg [hunk 2]
```diff
@@ -33,3 +33,38 @@ install_requires =
...
+    requests
+    beautifulsoup4
+    coverage>=3.7.0
+    black
+    flake8
+    mypy
+    types-requests
+    radicale
+
+[flake8]
```
Comment (weber-s) [inline setup.cfg:49]: Maybe we should keep setup.cfg only for release related stuff. And other python settings in pyproject.toml ?

Comment (weber-s) [reply setup.cfg:49]: For instance, there is 2 config for isort (here and in pyproject.toml)

File: tests/templates/dalec_tests/test-multiple-hours.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{% load dalec %}
```
Comment (weber-s) [inline tests/templates/dalec_tests/test-multiple-hours.html]: Why this test here?

File: tests/templates/dalec_tests/test-quarter.html
Comment (weber-s) [inline tests/templates/dalec_tests/test-quarter.html:1]: Why this test here?

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    requests
+    beautifulsoup4
+    coverage
+    radicale
+
+[testenv:qa]
+commands =
+    black --check ./
+    flake8
+    mypy -p dalec -p dalec_prime -p dalec_exemple
```
Comment (weber-s) [inline tox.ini]: Aren't you testing the base dalec here?

**Review by weber-s — COMMENTED** [review]
LGTM! I just have few non-blocking questions

File: tox.ini (hunk 4 above)
Comment (DylannCordel) [reply tox.ini]: done

File: tests/templates/dalec_tests/test-quarter.html
Comment (DylannCordel) [reply tests/templates/dalec_tests/test-quarter.html:1]: done

File: tests/templates/dalec_tests/test-multiple-hours.html (hunk 3 above)
Comment (DylannCordel) [reply tests/templates/dalec_tests/test-multiple-hours.html]: done

File: setup.cfg (hunk 2 above)
Comment (DylannCordel) [reply setup.cfg:49]: to do once the PR to add the pyproject.toml will be merged

**Comment by DylannCordel** [comment]
\o/ Looks good : tests ok with django main and dalec main. Once `dalec` will be released on pypi with version 0.2.0, we will need to restart those tests.

**Pull request merged**